packages/contracts-bedrock/.gitignore

```diff
@@ -6,3 +6,4 @@ coverage.out
 .deps
 deployments
```

Comment:
i think we do want to commit in the deployments directory besides `deployments/devnetL1` but fine for now